db/migrate/20180410104202_create_campaigns.rb

```diff
@@ -0,0 +1,16 @@
+class CreateCampaigns < ActiveRecord::Migration[5.1]
+  def change
+    create_table :campaigns do |t|
+      t.string :title
+      t.text :description
+      t.string :status
+      t.integer :current_amount
```

Comment:
All `"money fields"` should have `float` type.

Reply:
decimal :)

Reply:
or decimsl :)

Reply:
https://stackoverflow.com/a/66678